### ISSUE TITLE
Move project to new repo?

### ISSUE THREAD

**Issue by wmonecke** [rt2zz/redux-persist#1463]
I see that @rt2zz isn't active anymore. I have tried reaching out through multiple platforms without any response. Nonetheless, I think we are all grateful for his contributions and owe Zack a big thank you.

However, a lot of people are using this package and there are 70+ PRs open and countless issues that are quite critical.

I propose we move this package to a new "official" repo where all these PRs can be merged.

I am willing to create the repo and start merging stuff.

Is this something that you guys are interested in? Otherwise, I will just fork and do my stuff but the community wouldn't benefit as a whole.

**Comment by ZaidTryngo** [comment]
Please 🙏 Do this

**Comment by yeupanhmaj** [comment]
agree with you

**Comment by mxsxs2** [comment]
If that happens it could be a good idea to integrate encryption to the package too:
https://github.com/maxdeviant/redux-persist-transform-encrypt

**Comment by ghost** [comment]
> I see that @rt2zz isn't active anymore. I have tried reaching out through multiple platforms without any response. Nonetheless, I think we are all grateful for his contributions and owe Zack a big thank you.
> 
> However, a lot of people are using this package and there are 70+ PRs open and countless issues that are quite critical.
> 
> I propose we move this package to a new "official" repo where all these PRs can be merged.
> 
> I am willing to create the repo and start merging stuff.
> 
> Is this something that you guys are interested in? Otherwise, I will just fork and do my stuff but the community wouldn't benefit as a whole.

This or a fork would be appreciated.

**Comment by wmonecke** [comment]
@ckalika You mentioned you were taking over the repo - this is me trying one last time to see if we can get PRs merged. Would you be open to me joining as a maintainer? We could chat about this if you like.

**Comment by wmonecke** [comment]
I encourage anyone here to try [https://github.com/zewish/redux-remember](https://github.com/zewish/redux-remember) before jumping ship to another solution.

**Comment by UnexpectedLobster** [comment]
> I encourage anyone here to try https://github.com/zewish/redux-remember before jumping ship to another solution.

Is there any pros/con or any kind of comparison between those two projects (other than persist being unmaintained) ?

**Comment by wmonecke** [comment]
@PymZoR Not really, redux-persist has a HOC for when the state is being rehydrated that redux-remember does not have (but you can easily write).

IMO redux-remember is minimal and that makes it quite attractive.

**Comment by aqeelat** [comment]
@wmonecke I'll try to reach out to @ckalika on linkedin to bring his attention to this issue.

**Comment by ckalika** [comment]
Thanks @aqeelat for the ping, separately.

Hi guys.

I've not been active on this for a long while due to personal and professional responsibilities getting in the way, and that's entirely my bad.

I need to spend a bit of time cleaning things up here, but I agree with a lot of what you guys are saying.

If you guys can bear with me for a bit, I'll try to prioritise this more and see what we can get closed in the next little while.

**Comment by wmonecke** [comment]
@ckalika Hey there! 

Would you be open to adding another repo maintainer to the project? Having only one active maintainer in such a big repo could mean leaving PRs open for quite some time.

**Comment by clintonmedbery** [comment]
I agree with @wmonecke. I would create some issues and let whomever shows some initiative get a maintainer role.

**Comment by aqeelat** [comment]
I would suggest we first create a transparent CI/CD process requiring at least 2 devs to approve PRs, and prevent any individual maintainer from uploading release archives manually.Perhaps create multiple micro releases where after each PR is merged, a new release is created.This way, the trust barrier is significantly lower.Sent from my iPhoneOn May 10, 2024, at 6:13 AM, Clinton Medbery ***@***.***> wrote:﻿
I agree with @wmonecke. I would create some issues and let whomever shows some initiative get a maintainer role.

**Comment by gethari** [comment]
Is this project dead ?

**Comment by MDSADABWASIM** [comment]
A better choice would be to fork it and announce it here.

**Comment by mdemichele** [comment]
Hey everyone, I went ahead and made a new fork of the project here: https://github.com/mdemichele/redux-persist

Anyone is welcome to contribute, so feel free to reach out and I'll add you as a maintainer. I'm going to start by addressing the existing issues and merging in existing PR's. Then will try to do a new release once we have most of that out. We can see where it goes from there and start building up a community to make sure the project stays actively maintained in the future.

**Comment by Irashad** [comment]
@ckalika  , @wmonecke  hey, do you have any updates on this ?

**Comment by Irashad** [comment]
> Thanks [@aqeelat](https://github.com/aqeelat) for the ping, separately.
> 
> Hi guys.
> 
> I've not been active on this for a long while due to personal and professional responsibilities getting in the way, and that's entirely my bad.
> 
> I need to spend a bit of time cleaning things up here, but I agree with a lot of what you guys are saying.
> 
> If you guys can bear with me for a bit, I'll try to prioritise this more and see what we can get closed in the next little while.


2 years passing from your promise :)
